local/local.go

```diff
@@ -685,7 +685,6 @@ func defaultInspect(docker client.CommonAPIClient) (types.ImageInspect, error) {
 
 	return types.ImageInspect{
 		Os:           daemonInfo.OSType,
-		OsVersion:    daemonInfo.OSVersion,
```

Comment:
I think removing the default value for `OsVersion` is fine - and improvement actually. A blank value will generate more consistent images that any daemon can pull. The existing behavior is that a new `local` image - using the default `OsVersion` from daemon - could only be pulled by other daemons of equal or greater version.